pkg/agent/route/route_linux.go

```diff
@@ -416,6 +416,21 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet string, snatMa
 			"-j", iptables.MasqueradeTarget,
 		}...)
 	}
+
+	// For local traffic going out of the gateway interface, if the source IP does not match any
+	// of the gateway's IP addresses, the traffic needs to be masqueraded. Otherwise, we observe
+	// that ARP requests may advertise a different source IP address, in which case they will be
+	// dropped by the SpoofGuard table in the OVS pipeline. See description for the arp_announce
+	// sysctl parameter.
+	writeLine(iptablesData, []string{
+		"-A", antreaPostRoutingChain,
+		"-m", "comment", "--comment", `"Antrea: masquerade LOCAL traffic"`,
```

Comment:
Not sure which is more correct, the comment for the rule that marks local traffic is "Antrea: mark local output packets", could we make them consistent?

Reply:
I made it `LOCAL`. I didn't have a preference so I aligned with kube-proxy:
```
-A KUBE-XLB-FXIYY6OHUSNBITIX -m comment --comment "masquerade LOCAL traffic for default/my-service LB IP" -m addrtype --src-type LOCAL -j KUBE-MARK-MASQ
-A KUBE-XLB-FXIYY6OHUSNBITIX -m comment --comment "route LOCAL traffic for default/my-service LB IP to service chain" -m addrtype --src-type LOCAL -j KUBE-SVC-FXIYY6OHUSNBITIX
```